events/b_1_ipfs-impls.toml

```
@@ -97,9 +97,9 @@ description=""
 
 [[timeslots]]
 startTime="14:15"
-speakers=["@expede?"]
+speakers=["@expede","@matheus23"]
 title="WNFS"
```

Also, similar to the UCAN session title, maybe we expand the acronym once?

```suggestion
title="WebNative File System (WNFS)"
```